src/js/view/controls/components/timeslider.js

```diff
@@ -179,9 +175,6 @@ class TimeSlider extends Slider {
     }
 
     onPlaylistItem(model, playlistItem) {
-        if (!playlistItem) {
```

Comment:
@pajong Are we sure we don't need this check for `!playlistItem`? I looked back in the history and don't see the exact reason why this was put here though.

Reply:
It was there because we used to use `viewModel`, instead of `playerViewModel`. Now that we use the player view model, we always get `playlistItem`